packages/runtime-core/src/errorHandling.ts

```diff
@@ -114,7 +114,7 @@ export function handleError(
     if (appErrorHandler) {
       callWithErrorHandling(
         appErrorHandler,
-        null,
+        instance,
```

Comment:
This error is throw by `app.config.errorHandler` not instance , if pass `instance`, `callWithErrorHandling` will call `handleError` again, it's an infinite loop.